test/turbolence.spec.js

```diff
@@ -69,7 +69,7 @@ describe('Turbolence tests', () => {
   })
 
   it('crash that node, ensure nodeA continues going steady', (done) => {
-    nodeSpawn.kill('SIGKILL')
+    setTimeout(() => nodeSpawn.kill('SIGKILL'), 1000)
```

Comment:
Due to a longer node startup, another node can crash before we have the time to wrap the TCP socket with the first pull-stream, causing an error to be completely uncaught.

Consider: Should we always do .on('error') on TCP sockets? (And probably for all the other transports as well?)

@dignifiedquire @VictorBjelkholm